Firestore/core/src/util/firestore_exceptions.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_UTIL_FIRESTORE_EXCEPTIONS_H_
+#define FIRESTORE_CORE_SRC_UTIL_FIRESTORE_EXCEPTIONS_H_
+
+// Common C++ exception declarations across iOS and Android. These aren't a
+// public API surface.
+//
+// See exception.h for how to throw exceptions in a platform-agnostic way.
+
+#include <stdexcept>
```

Comment:
Seems like this should be `<exception>`, not `<stdexcept>`.

Reply:
Done. `<stdexcept>` made sense when the base type was `std::logic_error`.